skbase/base/_meta.py

```diff
@@ -207,7 +209,14 @@ def _get_params(
             ]
             out.update(named_objects_)
             for name, obj in named_objects_:
-                if hasattr(obj, method):
+                # checks estimator has the method we want to call
+                cond1 = hasattr(obj, methodd)
```

Comment:
for all objects in scope, they are present "if and only if", (public and private default), but I suppose it would be cleaner to check for the method that is actually called - you are right